Libs/DICOM/Core/ctkDICOMDatabase_p.h

```diff
@@ -174,23 +174,25 @@ class CTK_DICOM_CORE_EXPORT ctkDICOMDatabasePrivate
   bool openTagCacheDatabase();
   void precacheTags(const ctkDICOMItem& dataset, const QString sopInstanceUID);
 
-  // Return true if a new item is inserted
-  bool insertPatientStudySeries(const ctkDICOMItem& dataset,
+  // Return 1 if a new item is inserted
+  // Return 0 if a no item is inserted
+  // Return -1 if an error occurred
+  int insertPatientStudySeries(const ctkDICOMItem& dataset,
     const QString& patientID,
     const QString& patientsName,
     const QString& connectionName = "");
-  bool insertPatient(const ctkDICOMItem& dataset,
+  int insertPatient(const ctkDICOMItem& dataset,
     const QString& patientID,
     const QString& patientsName,
     int& databasePatientID);
-  bool insertConnectionName(const int& dbPatientID,
+  int insertConnectionName(const int& dbPatientID,
     const QString& connectionName);
-  bool updateConnections(const QString& dbPatientID,
+  int updateConnections(const QString& dbPatientID,
```

Comment:
Move out this "update" method from the list of "insert" methods and update its documentation.